spleeter_service/spleeter_service.py

```
@@ -0,0 +1,76 @@
+                response = {'status': 'error', 'message': str(e)}
+
+            # Clean up input file
+            os.remove(filename)
+
+            # Send response
+            self.send_response(200)
+            self.send_header('Content-type', 'application/json')
+            self.end_headers()
+            self.wfile.write(json.dumps(response).encode('utf-8'))
+
+def run_server(server_class=HTTPServer, handler_class=SeparatorHandler, port=5001):
```

Ruff (0.8.0)

Missing return type annotation for public function `run_server`

Add return type annotation: `None`

(ANN201)